pkg/api/v1beta1/dynakube/doc.go

```diff
@@ -0,0 +1,3 @@
+// Deprecated: This package is deprecated and will be removed in future versions.
```

Comment:
What about `v1beta2`? Should that also be deprecated and no longer used in the code?

It would not impact anyone that still uses those versions, since we have conversion, but the code should be all `v1beta3`

Reply:
done